### ISSUE TITLE
Is this an error in the CSS code?

### ISSUE THREAD

**Issue by ghost** [octobercms/october#4591]
I'm seeing this:

```css
*margin-right: .3em;
```

To me this should be this:

```css
margin-right: .3em;
```

It's being complied right now in the octobercms website and I've found it in this repo e.g.

https://github.com/octobercms/october/search?l=CSS&q=%2Amargin-right

Code lines:

https://github.com/octobercms/october/blob/fb2aa1730cc08e8563908b54b70e942bd2988f5e/modules/cms/widgets/assetlist/assets/css/assetlist.css#L90

https://github.com/octobercms/october/blob/fb2aa1730cc08e8563908b54b70e942bd2988f5e/modules/cms/widgets/assetlist/assets/css/assetlist.css#L127

https://github.com/octobercms/october/blob/4e92686c1a7de662840fa160b7348003f5ba817d/modules/backend/widgets/reportcontainer/assets/css/reportcontainer.css#L36

https://github.com/octobercms/october/blob/26173486d39cdf4312535ecf247e3d7a161713b4/modules/cms/assets/css/october.components.css#L64

https://github.com/octobercms/october/blob/26173486d39cdf4312535ecf247e3d7a161713b4/modules/cms/assets/css/october.components.css#L275

https://github.com/octobercms/october/blob/91d7b980a9abd299f0300b878b43410934a696b4/modules/backend/formwidgets/colorpicker/assets/css/colorpicker.css#L12

https://github.com/octobercms/october/blob/fb2aa1730cc08e8563908b54b70e942bd2988f5e/themes/demo/assets/vendor/font-awesome/css/font-awesome.css#L44

https://github.com/octobercms/october/blob/fb2aa1730cc08e8563908b54b70e942bd2988f5e/themes/demo/assets/vendor/font-awesome/less/mixins.less#L15

https://github.com/octobercms/october/blob/fb2aa1730cc08e8563908b54b70e942bd2988f5e/modules/system/assets/ui/less/icon.mixins.less#L15

p.s. Chrome is not liking it at all and it's actually stopping the `margin-right` from working and can be seen in the checkboxes when they are ticked, e.g. on this web page: https://octobercms.com/docs/ui/form

![image](https://user-images.githubusercontent.com/17784082/64202046-0f5ae500-ce88-11e9-86eb-5b199340443e.png)

- Gives 15

![image](https://user-images.githubusercontent.com/17784082/64202073-226db500-ce88-11e9-9182-df513819287d.png)

- Gives 3.6

Correct output value needs to be 3.6 by the way to match the rest of the labels.

p.s. If this is to fix Internet Explorer 7 surely this should be removed anyway. We should support IE11 only.

**Comment by w20k** [comment]
@ayumihamsaki it is correct, pretty much. Called `Holly Hack` met to fix CSS rule bugs in IE5-IE7 and is similar to the underscore hack. More info you could read here: https://web.archive.org/web/20150324181023/http://www.communitymx.com/content/article.cfm?page=2&cid=C37E0

**Comment by ayumihamsaki** [comment]
@w20k I was coding embed 8 bit microprocessors instead back then lol

Any reason why we should still support IE5 - IE7 now?

**Comment by w20k** [comment]
Lucky you, @ayumihamsaki 😄 IE5-IE6 - was a nightmare... Don't even want to remember. 
Nope, don't think we need it anymore. IE5-IE7 is dead.

**Comment by ayumihamsaki** [comment]
@w20k Yah I pity anyone working on really old IE versions, I think I started on IE9 and it was junk. Now we got Safari browser to give us nightmares - thanks apple for locking the operating system updates to certain levels on various apple devices. 😄

Will add it to my to-do list and create a pr sometime.

Thanks for the info! I was looking up CSS2 styles and thought it was to do with this at first: https://www.w3schools.com/cssref/sel_all.asp
